### PR TITLE
1a-feature-user-input-form-haroon-ali

### PR DESCRIPTION
**Pull request by haroon-ali-dev**
Created header form for user input for the first part of the application.

### CODE REVIEW

File: client/src/components/UserForm.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    pinCode: "",
+    phoneNumber: "",
+    qualification: "",
+    dateOfRelease: "",
+    caseStatus: ""
+  });
+
+  const handleChange = (e) => {
+    setInput({ ...input, [e.target.name]: e.target.value });
+  }
```
Comment (Miladepour) [inline client/src/components/UserForm.js:22]: ; missing semicolon here only.

**Review by Miladepour — APPROVED** [review]
Thank you @haroon-ali-dev, I've tested this and everything works perfectly.

**Comment by Miladepour** [comment]
@EneVladMihai @natyfromwonderland @miwsher
Can one of you review this pull request, please?

**Review by KptCnyt — APPROVED** [review]

**Review by baharbazargan62 — COMMENTED** [review]
that 's great.

**Review by baharbazargan62 — COMMENTED** [review]
that 's great.

**Review by baharbazargan62 — COMMENTED** [review]
review done.

**Pull request merged**